apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/R4CoverageResourceProviderIT.java

```diff
@@ -28,9 +28,12 @@
 public final class R4CoverageResourceProviderIT {
   private static final Logger LOGGER = LoggerFactory.getLogger(R4CoverageResourceProviderIT.class);
 
+  /**
+   * Ensures that {@link PipelineTestUtils#truncateTablesInDataSource()} is called before each test
+   * case.
```

Comment:
looks good but this comment is a little off since this will only run once before the entire suite

Reply:
Yeah let me fix that. I thought semantically it was passable, but I'll make it is more explicitly clear how many times it's called.